src/dir.targets

```diff
@@ -94,8 +94,18 @@
   <Import Project="$(ToolsDir)publishtest.targets" Condition="Exists('$(ToolsDir)publishtest.targets')" />
 
   <PropertyGroup>
+    <IsInnerLoopTestProject Condition="'$(IsInnerLoopTestProject)'==''">True</IsInnerLoopTestProject>
```

Comment:
I would default it the same way you are doing here. Just set it to be InnerLoop;OuterLoop if it is set to empty.